### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/poppy/commit/ad016e2ed75f84437bc050aed3d9d2b240207820#commitcomment-62588518

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/poppy/commit/a21aa4df7cd5a76cc08d16af2a531bfb550834ef

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

I intend to annotate this PR shortly. I'm happy to drop changes to files, drop specific terms, or split things as requested, although this may take some time.

### CODE REVIEW

File: docs/_themes/theme_webbpsf/static/bootstrap-astropy.css [hunk 1]
```diff
@@ -485,7 +485,7 @@ a.headerlink {
 a.headerlink:hover {
     background-color: #0069d6;
     color: white;
-    text-docoration: none;
+    text-decoration: none;
```
Comment (jsoref) [inline docs/_themes/theme_webbpsf/static/bootstrap-astropy.css:488]: This bug fix will probably result in a behavior change

File: docs/about.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ The Team
 -----------------
 
 POPPY is developed primarily by a team of astronomers at the `Space Telescope
-Science Insitute <http://www.stsci.edu/>`_, but is open to contributions from
+Science Institute <http://www.stsci.edu/>`_, but is open to contributions from
```
Comment (jsoref) [inline docs/about.rst:25]: Name

File: docs/about.rst [hunk 3]
```diff
@@ -22,7 +22,7 @@ The Team
...
 scientists and software developers around the world. Development takes place
 on Github at http://github.com/spacetelescope/poppy. See that page for the most up-to-date
```
Comment (jsoref) [inline docs/about.rst:27]: I usually include a commit to fix the brand `GitHub`, it appears I skipped that for this round.

File: docs/coronagraphs.rst [hunk 4]
```diff
@@ -108,13 +108,13 @@ Again we will compare the execution time with the FFT case.::
...
-        annFPM_fft_psf, annFPM_fft_interm = fftcoron_annFPM_osys.calc_psf(wavelen, display_intermediates=True,\
+        annFPM_fft_psf, annFPM_fft_interim = fftcoron_annFPM_osys.calc_psf(wavelen, display_intermediates=True,\
                                                                  return_intermediates=True)
         t1_fft = time.time()
         
         plt.figure()
         t0_mft = time.time()
-        annFPM_mft_psf, annFPM_mft_interm = matrixFTcoron_annFPM_osys.calc_psf(wavelen, display_intermediates=True,\
+        annFPM_mft_psf, annFPM_mft_interim = matrixFTcoron_annFPM_osys.calc_psf(wavelen, display_intermediates=True,\
                                                                      return_intermediates=True)
```
Comment (jsoref) [inline docs/coronagraphs.rst:118]: As the whitespace for the second line didn't line up consistently for these two calls, I didn't adjust it for this change.

File: docs/relnotes.rst [hunk 5]
```diff
@@ -389,7 +389,7 @@ improved upon in a future release.
 
  * Fix ConfigParser import (see `astropy/package-template#172 <https://github.com/astropy/package-template/pull/172>`_)
  * Fixes to formatting of ``astropy.units.Quantity`` values (`#171 <https://github.com/spacetelescope/poppy/issues/171>`_, `#174 <https://github.com/mperrin/poppy/pull/174>`_, `#179 <https://github.com/mperrin/poppy/pull/174>`_; @josephoenix, @neilzim)
- * Fixes to ``fftw_save_wisdom`` and ``fftw_load_wisdom`` (`#177 <https://github.com/spacetelescope/poppy/issues/177>`_, `#178 <https://github.com/mperrin/poppy/pull/178>`_; @mmecthley)
+ * Fixes to ``fftw_save_wisdom`` and ``fftw_load_wisdom`` (`#177 <https://github.com/spacetelescope/poppy/issues/177>`_, `#178 <https://github.com/mperrin/poppy/pull/178>`_; @mmechtley)
```
Comment (jsoref) [inline docs/relnotes.rst:392]: Leaving dangling references to accounts (or even references to the wrong account) can be fairly problematic.

File: poppy/dms.py [hunk 6]
```diff
@@ -391,7 +391,7 @@ def _get_surface_via_gaussian_influence_functions(self, wave):
 
         interpolated_surface = np.zeros(wave.shape)
 
-        crosstalk = 0.15  # amount of crosstalk on advancent actuator
+        crosstalk = 0.15  # amount of crosstalk on adjacent actuator
```
Comment (jsoref) [inline poppy/dms.py:394]: I'm not a domain expert, if this is a domain term, I'm sorry, and I'd been interested to learn about it. That said, the only Google hit for `advancent actuator` is this specific instance, so I'm guessing it's a typo. What it's a typo of, I'm much less confident.

File: poppy/instrument.py [hunk 7]
```diff
@@ -976,9 +976,9 @@ def _get_weights(self, source=None, nlambda=None, monochromatic=None, verbose=Fa
...
-            lambd = np.linspace(np.min(lrange), np.max(lrange), nlambda, endpoint=False) + (
-                    np.max(lrange) - np.min(lrange)) / (2 * nlambda)
+            points = np.linspace(np.min(lrange), np.max(lrange), nlambda, endpoint=False) + (
+                     np.max(lrange) - np.min(lrange)) / (2 * nlambda)
             filter_fn = scipy.interpolate.interp1d(wavelengths.to_value(units.m), throughputs, kind='cubic',
                                                    bounds_error=False)
-            weights = filter_fn(lambd)
-            return lambd, weights
+            weights = filter_fn(points)
+            return points, weights
```
Comment (jsoref) [inline poppy/instrument.py:984]: This is a significant divergence. I'm happy to drop it. But, it seems like `points` more accurately describes what the object is.

File: poppy/tests/test_core.py [hunk 8]
```diff
@@ -746,7 +746,7 @@ def test_CompoundOpticalSystem():
 
 # Tests for the inwave argument
 
-def test_inwave_fraunhoffer(plot=False):
+def test_inwave_fraunhofer(plot=False):
```
Comment (jsoref) [inline poppy/tests/test_core.py:749]: There were two misspellings of this, so I'm reasonably confident that they're misspellings.

File: poppy/tests/test_fft.py [hunk 9]
```diff
@@ -39,7 +39,7 @@ def test_fft_normalization():
     osys.add_pupil(pupil)
     osys.add_image() # null plane to force FFT
     osys.add_pupil() # null plane to force FFT
-    osys.add_detector(pixelscale=0.01, fov_arcsec=10.0) # use a large FOV so we grab essentially all the ligh
+    osys.add_detector(pixelscale=0.01, fov_arcsec=10.0) # use a large FOV so we grab essentially all the light
```
Comment (jsoref) [inline poppy/tests/test_fft.py:42]: Not terribly confident

File: poppy/tests/test_fresnel.py [hunk 10]
```diff
@@ -589,7 +589,7 @@ def test_fresnel_return_complex():
     # physical radius values
     M1_radius = 3. * u.m
     fl_M1 = M1_radius/2.0
-    # intermediary distances
+    # intermediate distances
```
Comment (jsoref) [inline poppy/tests/test_fresnel.py]: Hopefully ok?

File: poppy/tests/test_matrixDFT.py [hunk 11]
```diff
@@ -144,7 +144,7 @@ def test_MFT_fluxconsv_all_types(centering=None, **kwargs):
 
 
 
-def test_DFT_rect(centering='FFTSTYLE', outdir=None, outname='DFT1R_', npix=None, sampling=10., nlamd=None, display=False):
+def test_DFT_rect(centering='FFTSTYLE', outdir=None, outname='DFT1R_', npix=None, sampling=10., nlambda=None, display=False):
```
Comment (jsoref) [inline poppy/tests/test_matrixDFT.py:147]: I'm not particularly confident about these changes.

**Review by jsoref — COMMENTED** [review]
Most corrections were suggested by Google Sheets. I manually picked a couple. All fault lies with me.

I've tried to make sure everything makes sense, and I'm sorry if I've made mistakes (I'm human too, really!)

**Review by shanosborne — APPROVED** [review]
codecov patch is failing which is fine, merging anyway

**Pull request merged**